modules/signing_scriptworker/templates/dep-passwords-appsv.json.erb

```diff
@@ -0,0 +1,5 @@
+{
+    "project:mozilla:application-services:releng:signing:cert:dep-signing": [
+        ["https://autograph-external.prod.autograph.services.mozaws.net", "signingscript_focus_dep", "<%= scope.function_secret(["autograph_focus_dep_password"]) %>", ["autograph_focus"], "autograph"]
```

Comment:
TODO: do we want different credentials here for Autograph?

Reply:
I don't think we want the _focus_ dep key (is that what we're using for `android-components`?) but I don't think that we need an `application-services`-specific key. Can we use a generic "mozilla library" dep and release signing key?

Reply:
Good call, I pointed to the gpg default one,.